v22.1/backup-architecture.md

```diff
@@ -0,0 +1,96 @@
+
+- Test the connection to the storage bucket URL (`'s3://bucket'`).
+- Map out the directory to which the nodes will write.
+- Calculate the keys and time range of the backup data.
+- Determine the [leaseholder](architecture/overview.html#architecture-leaseholder) nodes for the keys to backup. 
+- Provide a plan to the leaseholder nodes.
+
+To map out the storage location's directory to which the nodes will write the data, the coordinator identifies the [type](backup-and-restore-overview.html#backup-and-restore-types) of backup. This determines the name of the new (or edited) directory to store the backup files in. For example, if there is a full backup existing in the target storage location, the upcoming backup will be [incremental](take-full-and-incremental-backups.html#incremental-backups) and therefore append to the full backup. For more information on how CockroachDB structures backups in storage, see [Backup collections](take-full-and-incremental-backups.html#backup-collections).
+
+### Key and time range resolution
+
+In this part of the resolution phase, the coordinator will calculate all the necessary spans of keys and their time range that the cluster needs to export for this backup. It divides the key spans based on which node is the range [leaseholder](architecture/overview.html#architecture-leaseholder) for that key. For this backup, each leaseholder node is responsible for:
+
+1. Asking the [storage layer](architecture/storage-layer.html) for the content of each key.
+1. Receiving the content from the storage layer. 
+1. Writing it to the backup storage location or [locality-specific location](take-and-restore-locality-aware-backups.html) (whichever locality best matches the node). 
+
+Since any node in a cluster can become the coordinator and all nodes could be responsible for exporting data during a backup, it is necessary that all nodes can connect to the backup storage location.
+
+## Export phase 
+
+Once the coordinator has provided a plan to each of the leaseholders that determines the backup data to process, the distributed export of the backup data begins.
+
+The backup export phase has two distinct parts:
+
+- [Distributed data exporting](#distributed-data-exporting)
+- [Metadata writing](#metadata-writing)
+
+### Distributed data exporting
+
+In the following diagram, **Node 2** and **Node 3** contain the leaseholders for the **R1** and **R2** [ranges](architecture/overview.html#architecture-range). Therefore, for the example backup job, these nodes will export the backup data to the specified storage location. While processing, the nodes write checkpoint progress to track their backup work. The coordinator node will aggregate the progress data into checkpoint files in the storage bucket. The checkpoint files provide a marker for the backup to resume after a retryable state, such as when it has been paused.
```

Comment:
"for the example backups job, the _backup processors on_ these nodes..."

also, the individual nodes do not write checkpoints; they return progress metadata to the coordinator who merges it to write the checkpoints as the next sentence says, so perhaps here it should just say "emit" or "return" progress information instead of "Write checkpoint" .